packages/backend-core/src/featureFlags/index.ts

```diff
@@ -36,7 +35,7 @@ export function isEnabled(featureFlag: string) {
 }
 
 export function getTenantFeatureFlags(tenantId: string) {
-  const flags = []
+  const flags = ["USER_GROUPS"]
```

Comment:
Was this a hack to get groups working locally by any chance? :laughing: Just forgot to remove it.
The easier way to do this is add the tenant feature flags to your `.env` file in the worker package like so:
```
TENANT_FEATURE_FLAGS=*:LICENSING,*:USER_GROUPS
```

Reply:
I'm a spoon, aye I'll remove this. I'm pretty sure I copied this from dev docs somewhere but I could be wrong.